### ISSUE TITLE
Comparison table accessibility issues

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.org#318]
- If the table has an empty header cell, it shouldn’t be marked up as an empty `<th>` element. Use an empty `<td>` instead.
- Table column headers (first row in bold) should be semantically marked up as `<th scope="col">`
- Table row headers (first cell in bold) should be semantically marked up as `<th scope="row">`
